ui/app/components/auth-form.js

```diff
@@ -121,7 +121,7 @@ export default Component.extend(DEFAULTS, {
   },
 
   firstMethod() {
-    const firstMethod = this.methodsToShow.firstObject;
+    const firstMethod = this.methodsToShow[0];
```

Comment:
No need to update, but I find destructuring to be a more elegant way to write this
```suggestion
    const [firstMethod] = this.methodsToShow;
```